chapters/zh/chapter2.md

```diff
@@ -20,14 +20,14 @@ id: 2
 
 ### Part 1
 
-- 在`nlp.vocab.strings`中查找字符串"cat"来得到哈希值。
+- 在`nlp.vocab.strings`中查找字符串"猫"来得到哈希值。
```

Comment:
In the slides for chapter 2.1, do you want to leave the vocab/hash examples in English, or should we try to use Chinese examples here?

Reply:
Right! Let me update chapter 2.1 with Chinese hash examples.